controllers/utils_test.go

```diff
@@ -18,121 +18,153 @@ package controllers
 
 import (
 	"errors"
+	"fmt"
+	"math/rand"
+	"time"
 
 	policiesv1 "github.com/kubewarden/kubewarden-controller/pkg/apis/policies/v1"
-	"github.com/onsi/gomega/types"
 
 	. "github.com/onsi/gomega" //nolint:revive
+	"github.com/onsi/gomega/types"
 
 	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
-	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	timeout      = 120 * time.Second
+	pollInterval = 250 * time.Millisecond
+)
+
 var (
-	someNamespace = corev1.Namespace{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "some-namespace",
-		},
-	}
 	templatePolicyServer = policiesv1.PolicyServer{
 		Spec: policiesv1.PolicyServerSpec{
-			Image:    "some-registry/some-policy-server:latest",
+			Image:    "ghcr.io/kubewarden/policy-server:latest",
 			Replicas: 1,
 		},
 	}
 	templateClusterAdmissionPolicy = policiesv1.ClusterAdmissionPolicy{
 		Spec: policiesv1.ClusterAdmissionPolicySpec{
 			PolicySpec: policiesv1.PolicySpec{
-				Module: "registry://some-registry/some/module:latest",
+				Module: "registry://ghcr.io/kubewarden/policies/echo",
```

Comment:
Can we use policies under the `test` path, as we use in the e2e tests? Just to avoid polluting the statistics by mistake.

Reply:
This is an [example](https://github.com/kubewarden/kubewarden-end-to-end-tests/blob/ac9b9958a00a9b284d8667708892c766c082a439/resources/policy-pod-privileged.yaml#L7) of the path used. We may need to push the echo policy there. But I don't see this as a big problem.

Reply:
Good point, I've changed it to use the pod-privilged policy, since it doesn't really matter which policy we use in this test.